itests/worker_test.go

```diff
@@ -585,82 +585,3 @@ waitForProof:
 	require.NoError(t, params.UnmarshalCBOR(bytes.NewBuffer(slmsg.Params)))
 	require.Equal(t, abi.RegisteredPoStProof_StackedDrgWindow2KiBV1_1, params.Proofs[0].PoStProof)
 }
-
-func TestWorkerPledgeExpireCommit(t *testing.T) {
-	kit.QuietMiningLogs()
-	_ = logging.SetLogLevel("sectors", "debug")
-
-	var tasksNoC2 = kit.WithTaskTypes([]sealtasks.TaskType{sealtasks.TTAddPiece, sealtasks.TTDataCid, sealtasks.TTPreCommit1, sealtasks.TTPreCommit2, sealtasks.TTCommit2,
-		sealtasks.TTUnseal, sealtasks.TTFetch, sealtasks.TTCommit1, sealtasks.TTFinalize, sealtasks.TTFinalizeUnsealed})
-
-	fc := config.DefaultStorageMiner().Fees
-	fc.MaxCommitGasFee = types.FIL(abi.NewTokenAmount(10000)) // 10000 attofil, way too low for anything to land
-
-	ctx := context.Background()
-	client, miner, worker, ens := kit.EnsembleWorker(t, kit.WithAllSubsystems(), kit.ThroughRPC(), kit.WithNoLocalSealing(true),
-		kit.MutateSealingConfig(func(sc *config.SealingConfig) {
-			sc.AggregateCommits = true
-		}),
-		kit.ConstructorOpts(
-			node.Override(new(*sealing.Sealing), modules.SealingPipeline(fc)),
-		),
-		kit.SplitstoreDisable(), // disable splitstore because messages which take a long time may get dropped
-		tasksNoC2)               // no mock proofs
-
-	ens.InterconnectAll().BeginMiningMustPost(2 * time.Millisecond)
-
-	e, err := worker.Enabled(ctx)
-	require.NoError(t, err)
-	require.True(t, e)
-
-	dh := kit.NewDealHarness(t, client, miner, miner)
```

Comment:
ditto; would be good to not lose this one too

Reply:
I mean this should get covered by Curio tests.